### ISSUE TITLE
TestFor::Code::TidyAll::Plugin isn't available to plugins on the CPAN

### ISSUE THREAD

**Issue by 2shortplanks** [houseabsolute/perl-code-tidyall#76]
There seems to be a standard way for testing plugins that are bundled with Code::TidyAll, using the TestFor::Code::TidyAll::Plugin module that's located inside `t/lib`.  However, plugins that are distributed as a separate CPAN distribution can't test using this code.

**Comment by autarch** [comment]
Good call. I'd be inclined to make it _not_ subclass `TestHelper::Test::Class` so you're not locked into `Test::Class::Most`. It could probably be rewritten as a `Role::Tiny` role, with guidance on calling the startup code as needed. We'd rename the `startup` method to something else. What do you think?

**Comment by oalders** [comment]
@2shortplanks ping
